apps/gauzy/src/app/pages/projects/components/project-edit/edit.component.ts

```
@@ -43,31 +44,57 @@ export class ProjectEditMutationComponent extends TranslationBaseComponent imple
 	 */
-	private _getEditProject() {
-		this.project$ = this._activatedRoute.data.pipe(
-			filter((data: Data) => !!data && !!data.project),
-			map(({ project }) => project),
-			tap((project) => (this.project = project)), // Assuming 'project' is a component property
-			untilDestroyed(this) // Automatically unsubscribes when the component is destroyed
-		);
+	private _getEditProject(data?: Data) {
+		const routeData = data || this._activatedRoute.snapshot.data;
+
+		if (routeData && routeData.project) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)